.github/workflows/functional-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Functional Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  functional:
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        iverilog-version: ["v10_3", "master"]
+        sim: ['icarus']
+        lang: ['verilog']
+        python-version: [3.8]
+        os: ['windows-latest']
+    env:
+      SIM: ${{matrix.sim}}
+      TOPLEVEL_LANG: ${{matrix.lang}}
+      PYTHON: ${{matrix.python-version}}
+      OS: ${{matrix.os}}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Anaconda ${{matrix.python-version}}
+      uses: conda-incubator/setup-miniconda@v1
+        with:
+          auto-update-conda: true
+          python-version: ${{matrix.python-version}}
+    - name: Set up cocotb build dependencies
+      run: |
+        conda install -c msys2 m2-base m2-make m2-autoconf m2-flex m2-bison m2-gperf m2w64-toolchain libpython
+    - name: Build and install Icarus
+      run: |
+        git clone https://github.com/steveicarus/iverilog.git -b ${{matrix.iverilog-version}}
+        cd iverilog
+        bash ./autoconf.sh
+        bash ./configure
```

Comment:
How come those two lines need `bash`?

Reply:
The first one does since it's not marked as executable. I can remove it from the second one.